src/core/ContentWindowController.h

```diff
@@ -75,6 +75,9 @@ class ContentWindowController : public QObject
     /** Resize the window. */
     Q_INVOKABLE void resize( QSizeF size, const WindowPoint fixedPoint = TOP_LEFT );
 
+    /** Resize the window relative to the given border */
+    Q_INVOKABLE void resizeRelative( const QPointF delta, const int border );
```

Comment:
Could we use the border enum here instead of an int? Without context, it is not possible to know what value to pass. Also, this class is unit tested, please add a unit test for the new function.